site/src/lib/playground.js

```diff
@@ -3,6 +3,7 @@ function createPlayground() {
     'use strict';
 
     var scriptTarget,
+        root,
```

Comment:
yes think could do something in the YAML but hadnt found a nice way yet and thought better to get examples in playground sooner rather than later